src/metadata/copy_obs/config.vsh.yaml

```diff
@@ -0,0 +1,68 @@
+name: copy_obs
+namespace: "metadata"
+description: |
+  Make a copy of an existing .obs column to another .obs column.
+authors:
+  - __merge__: /src/authors/dorien_roosen.yaml
+    roles: [ author ]
+argument_groups:
+  - name: "MuData Input"
+    arguments:
+      - name: "--input"
+        alternatives: ["-i"]
+        type: file
+        description: Input h5mu file
+        direction: input
+        required: true
+        example: input.h5mu
+      - name: "--modality"
+        type: string
+        default: "rna"
+        required: false
+      - name: "--obs_key_src"
```

Comment:
Would something like`--input_obs_key` and `--output_obs_key` also work for you?

The reasoning is that the files are now called `input` and `output` but the corresponding obs parameters are called `src` and `dest`.